addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html

```diff
@@ -12,6 +12,7 @@ <H1>Technology Detection Using Wappalyzer</H1>
 <ul>
 <li>It does not use the 'Global JavaScript variables' as these are difficult to test without a 'full' browser</li>
 <li>It does not not show the confidence - this is still todo</li>
+<li>It does not match technologies on the basis of DOM properties as some properties are set by JavaScript in the browser after response has passed through ZAP</li>
```

Comment:
Sorry I think I missed this last time:
```suggestion
<li>It does not match technologies on the basis of DOM properties as some properties are set by JavaScript in the browser after the response has passed through ZAP</li>
```